package.json

```diff
@@ -296,5 +296,8 @@
       "current node"
     ]
   },
-  "packageManager": "pnpm@10.10.0"
+  "packageManager": "pnpm@10.10.0",
+  "volta": {
+    "node": "22.16.0"
+  }
```

Comment:
is this part of the thread stuff? different pr would be good

Reply:
This was unintentional! Reverted.